backend/participant/tests.py

```diff
@@ -6,12 +6,14 @@
 from experiment.models import Block
 from session.models import Session
 from result.models import Result
+from django.core.management import call_command
 
 
 class ParticipantTest(TestCase):
 
     @classmethod
     def setUpTestData(cls):
+        call_command('bootstrap')
```

Comment:
Why is it necessary to call the bootstrap command? Wouldn't it be sufficient to directly call the `create_default_questions` function here?

Reply:
Yes, you're right, `create_default_questions` should be sufficient.